packages/sdk-middleware-auth/src/base-auth-flow.js

```diff
@@ -65,7 +66,7 @@ function executeRequest({
               const expirationTime = calculateExpirationTime(expiresIn)
 
               // Cache new token
-              tokenCache.set({ token, expirationTime, refreshToken })
+              tokenCache.set({ token, expirationTime, refreshToken }, clientId)
```

Comment:
I would like that we pass different options that might be useful for scoping the cached tokens and not just the `clientId`. The user can then freely decide how to construct the cache key.

For instance, possible options that we can pass are `clientId`, `projectKey`, `host`.

```js
tokenCache.set(
  { token, expirationTime, refreshToken },
  tokenCacheOptions
)
```

The `tokenCacheOptions` contains those values mentioned above.

Reply:
yes that sounds like more generic solution (I've tailored that to our implementation) 

I will also pass them to `get` otherwise the won't be a way to retrieve the token 😄